### DISCUSSION TITLE
error - uncaughtException: RangeError: Array buffer allocation failed

### DISCUSSION THREAD

**Discussion by FlyingTurkman** [vercel/next.js#43562] [Help]
With Next js 13 how can I fix this error?

**Comment by icyJoseph** [comment]
More context is needed. What are you trying to do?

**Reply by FlyingTurkman** [reply]
I am trying onAuthStateCheanged with firebase

**Reply by FlyingTurkman** [reply]
``export default function Layout({children}:{children:React.ReactNode}){
    onAuthStateChanged(auth,(user)=>{
        if(user){
            console.log('user',user.uid)
        }else{
            console.log('user',false)
        }
    })
    return(
        <main className='flex flex-row container p-3 items-start'>

**Reply by FlyingTurkman** [reply]
![image](https://user-images.githubusercontent.com/99084104/204825723-3c2f9274-10d0-4275-a5dc-89ff257ac22f.png)
